pkg/services/ngalert/state/historian/loki_http.go

```diff
@@ -11,25 +11,36 @@ import (
 
 const defaultClientTimeout = 30 * time.Second
 
+type LokiConfig struct {
+	Url            *url.URL
+	TenantID       string
+	TenantPassword string
+}
+
 type httpLokiClient struct {
 	client http.Client
-	url    *url.URL
+	cfg    LokiConfig
 	log    log.Logger
 }
 
-func newLokiClient(u *url.URL, logger log.Logger) *httpLokiClient {
+func newLokiClient(cfg LokiConfig, logger log.Logger) *httpLokiClient {
 	return &httpLokiClient{
 		client: http.Client{
 			Timeout: defaultClientTimeout,
 		},
-		url: u,
+		cfg: cfg,
 		log: logger.New("protocol", "http"),
 	}
 }
 
 func (c *httpLokiClient) ping() error {
-	uri := c.url.JoinPath("/loki/api/v1/status/buildinfo")
+	uri := c.cfg.Url.JoinPath("/loki/api/v1/labels")
```

Comment:
We changed the path we use for pinging, i.e. in Grafana Cloud, we don't expose this path. The `labels` path on the other side is mostly exposed.

Reply:
I'd suggest that we make sure to expose buildinfo for Loki endpoints.

Reply:
I'll make sure to reach out to understand if there is any specific reason why we don't expose it.